ddcz/templatetags/users.py

```
@@ -21,3 +22,19 @@ def level_star(user_profile, skin):
         ),
         "level": level,
     }
+
+
+@register.filter
+def nick_icon(nick):
+    try:
+        return UserProfile.objects.get(nick=nick).icon_url
+    except:
```

Please do not eat exceptions :)